### ISSUE TITLE
train-hyperparameter-tune-deploy-with-keras | Issue with callback log and history

### ISSUE THREAD

**Issue by rmikulec** [Azure/MachineLearningNotebooks#582]
**Describe the bug**
Currently is trying to access the accuracy using 'acc' as the key for both the log and history.history dicts.

Should be
- Line 88 :         run.log('Accuracy', log['accuracy'])
- Line 109 :       plt.plot(history.history['accuracy'], 'b-', label='Accuracy', lw=4, alpha=0.5)

Provide the following if applicable:
+ Python 3.6
+ Keras version: 2.3.0
+ Tensorflow version: 1.14.0

File link https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/training-with-deep-learning/train-hyperparameter-tune-deploy-with-keras/keras_mnist.py

**To Reproduce**
Attempt to run through the train-hyperparameter-tune-deploy-with-keras tutorial. Error will be logged on attempt to execute the job

**Comment by v-strudm-msft** [comment]
Thank you for reaching out to us.  We see this issue was not addressed and was posted some time ago. Our apologies. We will close it. Should you still need assistance with this or another issue, please re-post and we will respond promptly.





Thank   you for reaching out to us.  We see this issue was not addressed and was   posted some time ago. Our apologies. We will close it. Should you still need   assistance with this or another issue, please re-post and we will respond   promptly.
--


**
